src/components/Profile/UserProfile.jsx

```diff
@@ -0,0 +1,154 @@
+// React Imports
+import React, { useContext, useEffect, useState } from 'react';
+// Custom Hook Imports
+import { useSession } from '@hooks';
+// Material UI Imports
+import Box from '@mui/material/Box';
+import Button from '@mui/material/Button';
+import CheckIcon from '@mui/icons-material/Check';
+import ClearIcon from '@mui/icons-material/Clear';
+import EditIcon from '@mui/icons-material/Edit';
+// Context Imports
+import { SignedInUserContext } from '@contexts';
+// Component Inputs
+import ProfileImageField from './ProfileImageField';
+import ProfileInputField from './ProfileInputField';
+
+/**
+ * The UserProfile Component is a component that renders the user's profile on
+ * PASS
+ *
+ * @memberof Profile
+ * @name UserProfile
+ * @returns {React.JSX.Element} The UserProfile Component
+ */
+const UserProfile = () => {
```

Comment:
This `UserProfile` is very similar to the `ClientProfile` component. The user profile actually displays a bit more information, and IMO is formatted nicer. How about we just use this one `UserProfile` component, and then if the profile isn't for the logged-in user, we disable editing ability?

Reply:
Merged the two components together into a component called `ProfileComponent` to differentiate between this and the `Profile` page.

https://github.com/codeforpdx/PASS/assets/14917816/e1c1001c-3948-41db-a6a3-94118dd79405